src/modules/feature-modules/assessment/base/assessment-layout.component.ts

```diff
@@ -100,9 +100,8 @@ export class AssessmentLayoutComponent extends CoreComponent {
           { title: 'Innovation record', link: `/assessment/innovations/${currentRouteInnovationId}/record` },
           // { title: 'Action tracker', link: `/assessment/innovations/${currentRouteInnovationId}/action-tracker` },
         ];
-
+        this.leftSideBar.push({ title: 'Comments', link: `/assessment/innovations/${currentRouteInnovationId}/comments` });
```

Comment:
No need to push.

Just add it to this.leftSideBar = [ ...

Reply:
Okay